docker/Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM openjdk:11-jre-slim
+ARG TAG=11-jre-slim
```

Comment:
Do we want to use jre by default? Bundling the JDK tools might prove useful in several environments.

Reply:
I am not opinionated about this. My intention was to default on the "production" image. What do you think?

Reply:
I have changed to JDK version. Please have another look.